### ISSUE TITLE
normal Sampler Parameters for SDXL

### ISSUE THREAD

**Issue by drmbt** [cubiq/ComfyUI_essentials#92]
loving sampler parameters with grid, wish ive always had this. It doesn't work with non flux though, could you fork this node to work with  SDXL? Would also love params as a dict instead of a string, would be easier to pull a key from there

**Comment by cubiq** [comment]
yeah it's totally doable, I was thinking of doing one node that works with all models, but it's a lot of work :smile:... I'll think of something

**Comment by drmbt** [comment]
another RFE would just be an any input for PlotParameters to be able to accept arbitrary user determined information, could be appended, and params input could be optional

**Comment by cubiq** [comment]
that's a good idea

**Comment by drmbt** [comment]
If you do end up with one KSampler to rule them all, a huge request would be a params output with a breakout node to be able to further pipe any of that data downstream, for example to put the steps, cfg, sampler and inference run time in the filename. a dictionary out and the option to pull a single key would be very useful

**Comment by drmbt** [comment]
I know you're probably busy as hell right now, but with SD3.5 hitting the scene, it would be awesome if Sampler Parameters were adapted to support it

**Comment by drmbt** [comment]
any plans to support Stable Diffusion with your plotter?

**Comment by keyvez** [comment]
Hi Matteo, thanks for making the Flux Sampler Param node, it is super useful. If you do just a node that also supports SD instead of one that supports all, will that be easier? Also it would be great if that node saves/shows image output after every generation as opposed to after the whole batch. For example, I ram Flux Sampler Parameters, and it ran for 8 hours then ran out of memory, and there was no output, so it wasted the effort. Resuming would be a great feature too. I know this is a big ask, but just wondering what you think of these features.
